lib/web_ui/lib/pointer.dart

```diff
@@ -153,7 +153,23 @@ class PointerData {
   }
 }
 
+/// A function used to notify the engine about how a [PointerDataPacket] was used in the framework.
+///
+/// For example, can be used to [preventDefault] the source events on the web.
+typedef _PointerDataPacketAcknowledgeFn = void Function({
+  bool? preventDefault,
```

Comment:
Not sure if I want to add more parameters here that it'll affect the framework implementation. I think it won't, because the framework is not using this typedef for anything, and just directly calling `packet.acknowledge(preventDefault: true)`.